app/views/ShareView/index.tsx

```diff
@@ -42,6 +43,7 @@ interface IShareViewState {
 	loading: boolean;
 	readOnly: boolean;
 	attachments: IShareAttachment[];
+	compressionQuality: number;
```

Comment:
Let's control the state like this and create constants for the quality values instead of hardcoding.

```
const QUALITY_SD = 0.5;
const QUALITY_HD = 1;
```

```suggestion
	quality: 'sd' | 'hd';
```